R_program/Main.R

```diff
@@ -457,8 +475,11 @@ if(database_name!="dbWeatherData_Sagebrush_KP.v3.2.0.sqlite")
     #use with KP weather database
     YEARS<-c("50years","90years")
   }
+  
+  # prepend the downscaling method to all future conditions
+  temp <- paste0(downscaling.method, ".", rep(temp, each=length(downscaling.method))) #add (multiple) downscaling.method
 }
```

Comment:
since you have a comment above here (thanks), it looks like we can remove #add (multiple) downscaling.method